flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -201,8 +201,8 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .withDescription(
                             "A (semicolon-separated) list of vertex ids in hexstring for which to disable scaling. Caution: For non-sink vertices this will still scale their downstream operators until https://issues.apache.org/jira/browse/FLINK-31215 is implemented.");
 
-    public static final ConfigOption<Duration> SCALING_REPORT_INTERVAL =
-            autoScalerConfig("scaling.report.interval")
+    public static final ConfigOption<Duration> SCALING_EVENT_INTERVAL =
+            autoScalerConfig("scaling.event.interval")
                     .durationType()
                     .defaultValue(Duration.ofSeconds(1800))
                     .withDescription("Time interval to resend the identical event");
```

Comment:
```suggestion
                    .withDeprecatedKeys(
                            deprecatedOperatorConfigKey(
                                    "scaling.event.interval"))
                    .withDescription("Time interval to resend the identical event");
```

As this comment[1] mentioned: ` all config keys to work with the "old" syntax at least in the 1.7.0 release. `, so please update it, thanks!

Sorry for that, I didn't support the old key during decoupling autoscaler and kubernetes-operator.

https://github.com/apache/flink-kubernetes-operator/pull/686#discussion_r1369777698

Reply:
> As this comment[1] mentioned: `all config keys to work with the "old" syntax at least in the 1.7.0 release.`, so please update it, thanks!
> 
> Sorry for that, I didn't support the old key during decoupling autoscaler and kubernetes-operator.
> 
> [#686 (comment)](https://github.com/apache/flink-kubernetes-operator/pull/686#discussion_r1369777698)
updated.